### PR TITLE
[decoder/bounding_box] Support decoder for mediapipe palm detection

### PR DESCRIPTION
**Pull request by jihochu**
MP_PALM_DETECTION decoder for bounding box mode is presented. The model and
algorithm is based on mediapipe palm detection model and its
implementation.

Signed-off-by: Jiho Chu <jiho.chu@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: []Passed [ ]Failed [ *]Skipped

**How to evaluate:**

gst-launch-1.0 \
    v4l2src name=cam_src ! videoconvert ! videoscale ! video/x-raw,width=640,height=480,format=RGB,framerate=30/1 ! tee name=t \
    t. ! queue leaky=2 max-size-buffers=2 ! videoscale ! video/x-raw,width=192,height=192,format=RGB ! tensor_converter ! \
    tensor_transform mode=arithmetic option=typecast:float32,add:0.0,div:255.0 ! \
    tensor_filter framework=tensorflow-lite model=tflite_model/palm_detection_lite.tflite ! \
    tensor_decoder mode=bounding_boxes option1=mp-palm-detection option3=0.5 option4=640:480 option5=192:192 ! \
    compositor name=mix sink_0::zorder=2 sink_1::zorder=1 ! videoconvert ! ximagesink \
    t. ! queue leaky=2 max-size-buffers=10 ! mix.

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

File: ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c [hunk 1]
```diff
@@ -1112,6 +1365,68 @@ nms (GArray * results, gfloat threshold)
...
+    results = g_array_sized_new (FALSE, TRUE, sizeof (detectedObject), num_); \
+    for (d_ = 0; d_ < num_; d_++) { \
+      _type y_center, x_center, h, w; \
+      _type ymin, xmin; \
+      int y, x, width, height; \
+      detectedObject object; \
+      _type score = scores_[d_]; \
+      _type * box = boxes_ + boxbpi_ * d_; \
+      anchor *a = &g_array_index (data->anchors, anchor, d_); \
+      score = MAX(score, -100.0f); \
```
Comment (anyj0527) [inline ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:1399]: CI complained this for comparing float with other types (https://github.com/nnstreamer/nnstreamer/runs/6104768829?check_suite_focus=true#step:6:290).
May the `score` be float type could be better?

Comment (jihochu) [reply ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:1399]: thanks.
It changed to use `gfloat` type.

Comment (jihochu) [reply ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:1398]: thanks.
It changed to use gfloat type.

**Review by anyj0527 — APPROVED** [review]
LGTM 👍

**Pull request merged**